packages/dev/core/src/Rendering/IBLShadows/iblShadowsPluginMaterial.ts

```diff
@@ -39,6 +40,17 @@ export class IBLShadowsPluginMaterial extends MaterialPluginBase {
     public shadowOpacity: number = 1.0;
 
     private _isEnabled = false;
+    private _isColored = false;
+    public get isColored(): boolean {
+        return this._isColored;
+    }
+    public set isColored(value: boolean) {
+        if (this._isColored === value) {
+            return;
+        }
+        this._isColored = value;
+        this._markAllSubMeshesAsTexturesDirty();

```

Comment:
Is AsMiscDirty enough in this case ?

Reply:
I see a `markAllDefinesAsDirty` function. Is that preferable?

Reply:
Let s keep it as you have and I will check locally the best one to place it in